history-of-the-quakers-v4/original/07-book-7-chapter-1.adoc

```diff
@@ -51,21 +60,31 @@ And as it hath been our known principle to live peaceably under government,
 so we hope it will always be our practice, through God`'s assistance,
 to approve ourselves, with hearty affection, thy faithful and dutiful subjects.
 
+[.small-break]
+'''
+
 May the wonderful counsellor and great preserver
```

Comment:
This paragraph was indented more than the rest in the original and I'm not sure why. It seems to be okay aligned with the rest of the embed?